.github/workflows/pr-test.yaml

```diff
@@ -131,6 +135,11 @@ jobs:
                 exist: true,
                 title: /Test import/g,
                 state: /`success`[^`]+`success`/g
+              },
+              {
+                exist: false,
+                title: /Test run-all/g,
+                state: /`success`[^`]+`success`/g
```

Comment:
Why are we passing if the Test run-all comment doesn't exist? I would have expected a test that initializes multiple Terragrunt modules.

Reply:
Again, this was due to my misunderstanding. I could not get it to show up for the life of me but I needed to enable terragrunt and test for that as well. I will change that.